mxcube3/ui/components/Tasks/Characterisation.js

```diff
@@ -115,19 +115,14 @@ class Characterisation extends React.Component {
               </FieldsRow>
               <FieldsRow>
                 <InputField propName="exp_time" type="number" label="Exposure time (s)" />
-                <SelectField
-                  propName="beam_size"
-                  label="Beam size"
-                  list={this.props.apertureList}
-                />
+                <InputField propName="resolution" type="number" label="Resolution (Å)" />
```

Comment:
should we disabled by default beam_size instead

Reply:
We thought it was better to just remove it since its not at all wired at the backend. It's a feature we would like but there is no way of using it at the moment. The only thing it does right now is to take space in the interface